doc/network.tex

```diff
@@ -1328,18 +1346,20 @@ \subsection{Implementation of the Chain Consumer}
 as much as the other way around.
 However, two aspects of the protocol play in favour of the consumer here.
 \begin{itemize}
-  \item The protocol is basically consumer driver, i.e. the producer has simply no way to send unsolicited
+  \item The protocol is basically consumer driven, i.e. the producer has no way to send unsolicited
 data to the consumer (within the protocol).
   \item The consumer can verify the response data itself.
 \end{itemize}
 Here are some cases to consider:
 \begin{description}
-\item[\FindIntersect~Phase] The consumer and the producer play a number guessing game, the consumer can easily detect inconsistent behaviour.
+\item[\FindIntersect~Phase]
+  The consumer and the producer play a number guessing game, so the consumer can easily detect
+  inconsistent behaviour.
 \item[The producer replies with a \RollForward] The consumer can verify the block itself
   (with the help for the ledger layer)
   (The consumer may need to download the block first, if the protocol only sends block headers.)
```

Comment:
Lots of parenthesis here, there is no end-of-sentence `.` for the general bullet sentence.